src/platform/user/profile/vap-svc/actions/transactions.js

```diff
@@ -174,12 +174,11 @@ export function createTransaction(
 
       // If the request does not hit the server, apiRequest which is using fetch - will throw an error
       // it will not return back a transaction with errors on it
+      // TODO: use mock server locally instead of isVAProfileServiceConfigured
```

Comment:
ACTION ITEM: Is this a ticket somewhere?

Reply:
I will create a ticket around this. Good catch